src/app/story/storyrow/totaltimes/totaltimes.component.ts

```diff
@@ -0,0 +1,19 @@
+import { Component, Input } from '@angular/core';
+
+
+@Component({
+  selector: 'app-totaltimes',
+  templateUrl: './totaltimes.component.html',
+  styleUrls: ['./totaltimes.component.css', '../../storysidebar/storysidebar.component.css']
```

Comment:
Could you please refactor this to not rely on `storysidebar.component.css`? Since the two components are not related, it is a bit confusing. If you'd like to share styles between them, You could either move those classes to the global `styles.css`, or use a separate shared CSS file.

Reply:
It ended up that the css in `storysidebar.component.ts` wan't all needed, so I just altered it to what worked best and put it in the totaltimes css file. It let me take out some css in the storyrow css file as well.